corehq/sql_db/apps.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import absolute_import
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.db import connections
+
+
+class SqlDbAppConfig(AppConfig):
+    name = 'corehq.sql_db'
+
+    def ready(self):
+        """This gets run when Django starts"""
+        identify_standby_databases()
+
+
+# gets written when `identify_standby_databases` gets called or
+#   after Django starts.
+STANDBY_DATABASE_ALIASES = []
+
+
+def identify_standby_databases():
```

Comment:
you could just add `@memoized` to this and it will only get called the first time